packages/cli/tests/initializeReporters.test.ts

```diff
@@ -29,6 +31,7 @@ describe("initializeReporters", () => {
 
     expect(reporters.length).toBe(1);
     expect(reporters).toContainEqual(expect.any(ProgressReporter));
+    reporters.forEach((r) => r.cleanup?.());
```

Comment:
It would be safest to declare `let reporters: Reporter<LogStructuredData>[] | undefined` at the top level of the `describe`, then do this cleanup in `afterEach`. That way the cleanup runs even if the test fails.

Reply:
Sure, I updated it to work like that, and added a change file.